backend/tests/test_planner.py

```
@@ -50,6 +50,8 @@ def test_generate_plan_no_api_key():
         assert "no API key" in plan.description 
         assert plan.cluster_id == "c1"
 
+import pytest
```

Ruff (0.14.10)

Module level import not at top of file

(E402)